src/Integration.Vsix/SonarLintTagger/IssueMarkerFactory.cs

```diff
@@ -22,11 +22,11 @@
 using Microsoft.VisualStudio.Text;
 using SonarLint.VisualStudio.Core.Analysis;
 
-namespace SonarLint.VisualStudio.Integration.Vsix.Helpers
+namespace SonarLint.VisualStudio.Integration.Vsix
 {
-    internal class IssueConverter : IIssueConverter
+    internal class IssueMarkerFactory : IIssueMarkerFactory
```

Comment:
Feel free to suggest an alternative name. I'm not particularly happy with the new name, but it's less misleading than the old one.

Reply:
Hm, that's tough. The class is responsible for calculating the span, and then returning an object containing the issue+span, right?
Maybe it should do just the calculation, and become `IIssueSpanCalculator`, and have `TextBufferIssueTracker` call it and create IssueMarker. But I'm not sure it's really worth the change.